twilight-lavalink/README.md

```diff
@@ -13,12 +13,26 @@ handle sending voice channel updates to Lavalink by processing events via
 the [client's `process` method][`Lavalink::process`], which you must call
 with every Voice State Update and Voice Server Update you receive.
 
+A breakdown of how this functions:
+- The client is [`Lavalink`](crate::client::Lavalink) that forwards the required events from Discord.
+    - We read the [Voice State and Voice Server Updates](https://discord.com/developers/docs/topics/gateway-events#voice) from discord to format the data to send to a Lavalink `VoiceUpdate` Event.
+    - There is a lower level [node](crate::node) that processes this for you. It isn't recommended to use this but rather the lavalink struct with the players. If you don't find functionality please open up and issue to expose what you need.
+- You send the client an [outgoing event](crate::model::outgoing). These include play, pause, seek, etc. You send these through the [player](crate::player) that is attached to Lavalink.
+- If you want to search or load you need to create a http client currently and then you can use [these helpers functions](crate::http#functions) to generate the http uri and body to send over your http client. you will then get response you can deserialize as json into the structs in the [http module](crate::http).
```

Comment:
Could this information be better integrated into the rest of the README? Perhaps as comments in the example below?

Reply:
yea I can see how this might not be super helpful. I moved these comments to the basic bot example since that was a better example of where we are using these. For me it was helpful to have links to what was being used.

When I first came across the library I was very confused by what was what. Hopefully this helps someone in the future.